strawberry/channels/handlers/http_handler.py

```diff
@@ -247,6 +277,15 @@ async def get_context(
     async def get_sub_response(self, request: ChannelsRequest) -> TemporalResponse:
         return TemporalResponse()
 
+    async def create_multipart_response(
+        self,
+        request: ChannelsRequest,
+        stream: Callable[[], AsyncGenerator[str, None]],
+        sub_response: TemporalResponse,
+    ) -> MultipartChannelsResponse:
+        # TODO: sub response
+        return MultipartChannelsResponse(stream=stream)
```

Comment:
Taking a look at how `sub_response` is used today, shouldn't we be using it for status code and headers?

Reply:
yup, I'll try to add this